examples/quantization/brevitas/quantize_llm.py

```diff
@@ -95,40 +107,36 @@
     nsamples=128,
     seqlen=args.seqlen,
     split="validation",
-    device="cuda:0" if not args.cpu_offload else None,
+    device=args.device if not use_accelerate else None,
 )
 
-perplexity = compute_perplexity(
-    quantizer.model, validation_dataset, context_length=args.seqlen // 2, tokenizer=tokenizer
-)
+model = quantizer.model
 
+# Evaluation of the non-quantized model.
+if use_accelerate:
+    model = offload_model(model, qconfig.gpu_device_map, qconfig.cpu_device_map)
+perplexity = compute_perplexity(model, validation_dataset, context_length=args.seqlen // 2, tokenizer=tokenizer)
+if use_accelerate:
+    remove_hooks(model)
 print(f"Perplexity (original model): {perplexity}")
 
 quantized_model = quantizer.quantize(qconfig, calibration_dataset)
 
 # Evaluation of the quantized model.
-perplexity = compute_perplexity(
-    quantized_model, validation_dataset, context_length=args.seqlen // 2, tokenizer=tokenizer
-)
+if use_accelerate:
+    quantized_model = offload_model(quantized_model, qconfig.gpu_device_map, qconfig.cpu_device_map)
+perplexity = compute_perplexity(quantized_model, validation_dataset, context_length=args.seqlen // 2, tokenizer=tokenizer)
+if use_accelerate:
+    remove_hooks(model)
```

Comment:
Same, I don't think the controlflows are necessary (except the last one which is explicitely for the ONNX export, and should probably be moved below)